### DISCUSSION TITLE
URL containing "%2f" not re-directed

### DISCUSSION THREAD

**Discussion by OlivierBOEL** [OpenIDC/mod_auth_openidc#1164] [General]
Hi,

Sorry if this has been already discussed but I noticed that any URL containing "%2f" is not re-directed to OpenID provider bur, rather, processed by web server (which, usually, shows 404 as the file does not exist).
Did anyone else notice/experience this?

Thanks,

Olivier

**Comment by zandbelt** [comment]
see https://httpd.apache.org/docs/2.4/mod/core.html#allowencodedslashes

**Reply by OlivierBOEL** [reply]
Thanks!
